x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -394,7 +394,7 @@ export const RulesList = ({
     if (lastRunOutcomeFilter) {
       updateFilters({ filter: 'ruleLastRunOutcomes', value: lastRunOutcomeFilter });
     }
-  }, [lastResponseFilter]);
+  }, [lastRunOutcomeFilter]);
```

Comment:
Shouldn't ESLint have caught this? It seems the exhaustive-deps rule is not kicking in here.

Reply:
Good catch, looks like we have a legacy file wide ```/* eslint-disable react-hooks/exhaustive-deps */``` at the top of the file.

I will create an issue to remove it